scripts/update-spectrum-css.js

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env node
+
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { readFileSync, writeFileSync } from 'fs';
+import latestVersion from 'latest-version';
```

Comment:
Rather than pulling in a new dependency (latestVersion is a great package though it also pulls in packageJson with it), what would you think of running exec: `yarn info ${packageName} dist-tags.latest --json` this can be easily parsed for the value of data which holds the latest version number.  It lets us leverage yarn and exec rather than any new packages.  Just an option - I do think latestVersion is a nice clean syntax though.

Reply:
Initially, because I did not know this command. Currently, because I like not having to go back and forth between the subprocess and easier. I look at this more after your other notes.